cosi/mod.go

```diff
@@ -35,12 +28,17 @@ type Message interface {
 	GetHash() []byte
 }
 
+// Actor is the listener of a collective signing instance. It provides a
+// primitive to sign a message.
+type Actor interface {
```

Comment:
note for later: add `Actor` in the list of terminologies. All those terminologies can be confusing at first: Actor, Players, Verifier, ect...

Reply:
Yes, I'll add it now.